src/components/Footer/Footer.tsx

```diff
@@ -149,7 +151,11 @@ const StyledHr = styled.hr`
   background: linear-gradient(to right, ${constants.primaryTextColor}, rgba(0, 0, 0, 0));
 `;
 
-const Footer = ({ strings }) => (
+interface FooterProps {
+  strings: O.Path<AppState, ['app', 'strings']>;
```

Comment:
what does this look like without the O library? I think we can just use a `{ [key: string]: value: string }` type for this?

Reply:
Something like `AppState['app']['strings']` should work, I can actually remove ts toolbelt, added it while fighting with eslint to setup parser properly